README.md

```diff
@@ -32,18 +32,54 @@ Cordova Android is an Android application library that allows for Cordova-based
 
 ## Requirements
 
-- Java Development Kit (JDK) 11
-- [Android SDK](https://developer.android.com/)
+* Java Development Kit (JDK) 11
+* [Android SDK](https://developer.android.com/)
 
 ## Cordova Android Developer Tools
 
 Use the [Cordova command-line tool](https://www.npmjs.com/package/cordova) to create projects and install plugins.
 
+However, the following scripts can be used instead:
+
+    ./bin/create [path package activity] ... creates the ./example app or a cordova android project
+    ./bin/check_reqs ....................... checks that your environment is set up for cordova-android development
+    ./bin/update [path] .................... updates an existing cordova-android project to the version of the framework
+
+These commands live in a generated Cordova Android project. Any interactions with the emulator require you to have an AVD defined.
+
+    ./cordova/clean ........................ cleans the project
+    ./cordova/build ........................ calls `clean` then compiles the project
+    ./cordova/log   ........................ streams device or emulator logs to STDOUT
+    ./cordova/run   ........................ calls `build` then deploys to a connected Android device. If no Android device is detected, will launch an emulator and deploy to it.
+    ./cordova/version ...................... returns the cordova-android version of the current project
```

Comment:
Eventhough this script still exists, it most likely will be removed in the future.

The reason why the script still exists today, is because the CLI command `cordova platforms` calls this script to disiplay the installed platform version.

In the future, the CLI will just call directly to the platform's API to return the version.

If you decide to exclude adding this, then the entire section is no longer needed so you can also remove the opening:

> These commands live in a generated Cordova Android project. Any interactions with the emulator require you to have an AVD defined.

Reply:
Changes on Readme moved to PR #1308 ; and I removed this block from my commits